src/main/scala/beam/agentsim/agents/ridehail/OutOfServiceVehicleManager.scala

```diff
@@ -61,7 +61,7 @@ class OutOfServiceVehicleManager(
       .rideHailAgent
 
     rideHailAgent.tell(
-      ModifyPassengerSchedule(passengerSchedules.get(vehicleId).get),
+      ModifyPassengerSchedule(passengerSchedules(vehicleId)),
```

Comment:
I'd almost rather have the explicit get as it should be a flag for refactoring - where an Option might be wanted here?